modulefiles/GDAS/hera.lua

```diff
@@ -8,41 +8,60 @@ local pkgNameVer = myModuleFullName()
 
 conflict(pkgName)
 
-local jedi_opt = '/scratch1/NCEPDEV/jcsda/jedipara/opt/modules'
-setenv('JEDI_OPT', jedi_opt)
-local jedi_core = pathJoin(jedi_opt, 'modulefiles/core')
-prepend_path("MODULEPATH", jedi_core)
-
+prepend_path("MODULEPATH", '/scratch1/NCEPDEV/global/spack-stack/spack-stack-v1/envs/skylab-2.0.0-intel-2021.5.0/install/modulefiles/Core')
 prepend_path("MODULEPATH", '/scratch1/NCEPDEV/da/python/opt/modulefiles/stack')
 
-load("cmake/3.20.1")
-load("git-lfs/2.11.0")
-
-load("jedi-intel/2020.2")
-load("szip/2.1.1")
-load("zlib/1.2.11")
+-- below two lines get us access to the spack-stack modules
+load("stack-intel/2021.5.0")
+load("stack-intel-oneapi-mpi/2021.5.1")
+-- JCSDA has 'jedi-fv3-env/1.0.0', but we should load these manually as needed
+load("cmake/3.22.1")
+load("curl/7.29.0")
+load("gettext/0.19.8.1")
+load("libunistring/0.9.10")
+load("libidn2/2.3.0")
+load("pcre2/10.39")
+load("zlib/1.2.12")
+load("git/2.37.0")
+load("pkg-config/0.27.1")
+load("hdf5/1.12.1")
+load("parallel-netcdf/1.12.2")
+load("netcdf-c/4.8.1")
+load("nccmp/1.9.0.1")
+load("netcdf-fortran/4.5.4")
+load("parallelio/2.5.7")
+load("wget/1.14")
+load("boost/1.78.0")
+load("bufr/11.7.1")
+load("git-lfs/2.10.0")
+load("ecbuild/3.6.5")
+load("openjpeg/2.3.1")
+load("eccodes/2.27.0")
+load("eigen/3.4.0")
+load("openblas/0.3.19")
+load("eckit/1.19.0")
+load("fftw/3.3.10")
+load("fckit/0.9.5")
+load("fiat/1.0.0")
+load("ectrans/1.0.0")
+load("atlas/0.30.0")
+load("gsibec/1.0.5")
+load("gsl-lite/0.37.0")
+load("libjpeg/2.1.0")
+load("krb5/1.15.1")
+load("libtirpc/1.2.6")
+load("hdf/4.2.15")
+load("jedi-cmake/1.4.0")
+load("libpng/1.6.37")
+load("libxt/1.1.5")
+load("libxmu/1.1.2")
+load("libxpm/4.11.0")
+load("libxaw/1.0.13")
 load("udunits/2.2.28")
-load("gsl_lite/0.37.0")
-load("jedi-impi/2020.2")
-
-load("hdf5/1.12.0")
-load("pnetcdf/1.12.1")
-load("netcdf/4.7.4")
-
-load("boost-headers/1.68.0")
-load("eigen/3.3.7")
-load("bufr/noaa-emc-11.5.0")
-
-load("nccmp/1.8.7.0")
-load("pio/2.5.1-debug")
-
-load("ecbuild/ecmwf-3.6.1")
-load("eckit/ecmwf-1.18.2")
-load("fckit/ecmwf-0.9.5")
-load("atlas/ecmwf-0.29.0")
-load("nco/4.9.1")
```

Comment:
@CoryMartin-NOAA , nco is missing, that one of the reasons for the soca test failures.